### ISSUE TITLE
Copy_Both_Response is missing OverallFormat byte when Encoded

### ISSUE THREAD

**Issue by masterkidan** [jackc/pgproto3#26]
Hello, 
It looks Copy_Both_Response.go is missing the overallFormat byte as part of the first byte in the response. If you were to look at the decode method it is fairly obvious: 
```
       overallFormat := buf.Next(1)[0]

	columnCount := int(binary.BigEndian.Uint16(buf.Next(2)))
	if buf.Len() != columnCount*2 {
		return &invalidMessageFormatErr{messageType: "CopyBothResponse"}
        }
```
	
But in Encode we append `dst = pgio.AppendUint16(dst, uint16(len(src.ColumnFormatCodes)))` without appending the `src.OverallFormat` first.

**Comment by masterkidan** [comment]
I've opened #25 to address this.

**Comment by jackc** [comment]
Fixed in #25.
